interface/wx/event.h

```diff
@@ -3318,6 +3318,59 @@ class wxDisplayChangedEvent : public wxEvent
 };
 
 
+
+/**
+    @class wxDPIChangedEvent
+
+    Event sent when the resolution (measured in dots-per-inch, or DPI) of the
+    monitor a window is on changes.
+
+    The event is sent to each wxTopLevelWindow affected by the change, and all
+    its children recursively. For example, this event is sent to the window
+    when it is moved, by the user, from a display using some DPI value to
+    another display using a different DPI value. It also sent to all program
+    windows on the given display if its DPI changes due to a change in the
+    system settings.
+
+    Currently this event is generated by wxMSW port if only and only if the
+    MSW application runs under Windows 10 v1607 or later and is marked as being
```

Comment:
The description describes behaviour of "PerMonitorV2", which according to the link below is supported only since Windows 10 version **1703** (Creator's Update). OTOH, V1 which doesn't send the message to child controls and IIRC does not even scale the NC area properly, is available since Windows 8.1....

Reply:
Thanks for noticing. It should indeed be 1703.
I chose to require V2 because V1 (or `true/pm`) misses a lot of functionality. Non-client area scaling was possible in 1607 (you had to enable it manually with `EnableNonClientDpiScaling`) but it did not scale GUI elements like checkboxes and radio buttons.